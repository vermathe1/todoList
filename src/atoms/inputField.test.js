import React from 'react'
import renderer from 'react-test-renderer'
import InputField from './InputField.js'

describe('<InputField />', () => {
  it(' matches the snapshop', () => {
    const tree = renderer.create(<InputField />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})