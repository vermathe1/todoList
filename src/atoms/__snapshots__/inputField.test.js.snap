// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputField />  matches the snapshop 1`] = `
<input
  autoFocus={true}
  defaultValue=""
  id=""
  onBlur={[Function]}
/>
`;
